driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/unified/ChangeStreamsTest.java

```diff
@@ -36,8 +35,11 @@
 public final class ChangeStreamsTest extends UnifiedReactiveStreamsTest {
 
     private static final List<String> ERROR_REQUIRED_FROM_CHANGE_STREAM_INITIALIZATION_TESTS =
-            Collections.singletonList(
-                    "Test with document comment - pre 4.4"
+            Arrays.asList(
+                    "Test with document comment - pre 4.4",
+                    "Change Stream should error when an invalid aggregation stage is passed in",
+                    "The watch helper must not throw a custom exception when executed against a single server topology, "
+                            + "but instead depend on a server error"
```

Comment:
Maybe these two tests will succeed once we include https://github.com/mongodb/specifications/commit/dbfbc0f4f10234b1fa4f3f216b0919c68e4607bb in the PR as per this [comment](https://jira.mongodb.org/browse/JAVA-4127?focusedCommentId=4498366&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4498366).

Reply:
They still don't succeed.  I'm not sure the impetus for https://github.com/mongodb/specifications/commit/dbfbc0f4f10234b1fa4f3f216b0919c68e4607bb, but it doesn't surprise me that the change in that commit wouldn't have an affect on these test failures.